packaging/package-indexer/index-packages.py

```diff
@@ -125,6 +131,8 @@ def construct_indexers(cfg: Config, args: dict) -> List[Indexer]:
     indexed_remote_storage_synchronizer = cfg.create_indexed_remote_storage_synchronizer(suite)
     cdn = cfg.create_cdn(suite)
 
+    gpg_key_passphrase = stdin.read() if args["gpg_key_passphrase_from_stdin"] else None
```

Comment:
We could ask(prompt) for the passphrase in this case and stop reading after the first newline.

Reply:
Good idea, but I would leave that for the future, when the need arises.